frontend/src/store/client-settings.ts

```diff
@@ -52,6 +61,39 @@ class ClientSettingsStore extends SyncedStore<ClientSettingsState> {
     return this._state.darkMode;
   }
 
+  public set subtitleAppearance(newVal: ClientSettingsState['subtitleAppearance']) {
+    this._state.subtitleAppearance = newVal;
+  }
+
+  public get subtitleAppearance(): ClientSettingsState['subtitleAppearance'] {
+    return this._state.subtitleAppearance;
+  }
+
+  /**
+   * CSS Style Properties for subtitles 
+   */
+  public get subtitleStyle(): CSSProperties {
+    const strokeStyle = {
+      WebkitTextStrokeColor: "black",
+      WebkitTextStrokeWidth: "7px",
+      textShadow: "2px 2px 15px black",
+      paintOrder: "stroke fill"
+    }
+
+    return {
+      fontFamily: `${this.subtitleAppearance.fontFamily}, ${FALLBACK_SUBTITLE_FONT} !important`,
+      fontSize: `${this.subtitleAppearance.fontSize}em`,
+      marginBottom: `${this.subtitleAppearance.positionFromBottom}vh`,
+      backgroundColor: this.subtitleAppearance.backdrop ? 'rgba(0, 0, 0, 0.5)' : 'transparent',
+      padding: '10px',
+      color: 'white',
+      /**
+       * Unwrap stroke style if stroke is enabled
+       */
+      ...(this.subtitleAppearance.stroke && strokeStyle)
+    }
+  }
```

Comment:
Why not do something like this? https://vuejs.org/api/sfc-css-features.html#v-bind-in-css

Reply:
This was done for reuse-ability since these same CSS properties are used in the settings page and in the track component.

Reply:
@seanmcbroom Can't we use the SubtitleTrack component to display in settings as well? So we keep CSS with CSS. That way we don't also need the webkit prefixes, since those are handled by lightningcss at bundle time.

A boolean prop ``settings`` or passing a custom slot could do the trick maybe?